sdk/src/main/java/co/omise/android/ui/AuthorizingPaymentActivity.kt

```diff
@@ -265,8 +265,12 @@ class AuthorizingPaymentActivity : AppCompatActivity() {
         val resultIntent = Intent().apply {
             putExtra(EXTRA_AUTHORIZING_PAYMENT_RESULT, Failure(throwable))
         }
-        setResult(Activity.RESULT_OK, resultIntent)
-        finish()
+        if(throwable.message == "3DS2 initialization failed"){
+            setupWebView()
+        }else{
+            setResult(Activity.RESULT_OK, resultIntent)
+            finish()
+        }
```

Comment:
Let's add spacing around `if` and `else`. In other places, `if` and `else` has spacing.

Reply:
@aashishgurung @AnasNaouchi can we set a standard formatter for Java?

Reply:
I found [checkStyle](https://github.com/checkstyle/checkstyle)

Reply:
Great. Thanks @aashishgurung